pkgs/tools/archivers/torrent7z/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, fetchpatch, lib, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "torrent7z";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "BubblesInTheTub";
+    repo = pname;
+    rev = "c982324b0af6094e7e28b106f50cb8bbec70d227";
```

Comment:
Why are we not using the git tag?

Reply:
The base 1.3 release is broken on latest gcc and requires some patches to build. It has been fixed since but there's no new release pushed on Github.

Reply:
If it's a newer commit we usually try to use `unstable-yyyy-mm-dd` format for the version. It's confusing for the user if software claims to be version 1.3 but in reality newer commits might have more features, breaking changes, and/or a different UX.


I think it would be ok if the commits in between the commit and the tag only fix compilation, but adding a comment then would be appreciated.

Alternatively only fetching the patch (the url of the commit on github with .patch appended) that fixes compilation and then adding it to the patches attribute will work. That assumes the patch will apply without the other commit that happened in between